cmd/celestia/main.go

```diff
@@ -5,9 +5,16 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/cmd/celestia/bridge"
+	"github.com/celestiaorg/celestia-node/cmd/celestia/full"
+	"github.com/celestiaorg/celestia-node/cmd/celestia/light"
 )
 
 func init() {
+	bridgeCmd := bridge.NewCommand(bridge.WithSubcommands())
+	lightCmd := light.NewCommand(bridge.WithSubcommands())
+	fullCmd := full.NewCommand(bridge.WithSubcommands())
```

Comment:
@tuxcanfly i'm not quite following why we pass `bridge.WithSubCommands()` to the various different node types? Can you help me figure out what is happening here please?

Reply:
Looks like a typo

Reply:
Yep, fixed with move to main 👍🏽